.github/workflows/test.yml

```diff
@@ -209,13 +209,15 @@ jobs:
           image-output: '/tmp/root.img'
 
       - name: Configure AWS Credentials
+        if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
Maybe leave a quick comment as to why this is gated.

Reply:
Added comment